client/src/components/purchase.js

```diff
@@ -1,43 +1,75 @@
 import React from 'react';
+import ReviewStars from './reviewStars.js';
+import Stylepicker from './stylepicker.js';
 
 class Purchase extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
-
+      //May need refactoring if there is no state to take care of.
     };
   }
 
   render() {
     return (
       <div id="product-purchase">
-          <div id="product-purchase-price">FRIHETEN
-              Corner sofa-bed with storage, Skiftebo dark gray
-              $799.00
-              ★★☆☆☆ (222)</div>
-          <div id="product-purchase-stylepicker">
-              <button id="stylepicker-modaler-button">Cover Skiftebo dark grey</button>
-              <div id="stylepicker-variants">
-                {
-                  this.props.product.variants.map( variant => {
-                    return (
-                      <a key={variant._id} href={variant.linkUrl} className="stylepicker-variant-item">
-                        <div className="variant-image-container">
-                          <img src={variant.imageUrl} className="variant-image"></img>
-                        </div>
-                      </a>
-                    );
-                  })
-                }
+        <div id="product-purchase-price">
+            <div className="price-container">
+              <div className="price-infobox inline-block">
+                <div className="price-name">{this.props.product.name}</div>
+                <div className="price-description">{this.props.product.shortName + ((this.props.product.variantProduct) ? ',' : '')}</div>
+                <div className="purchase-text">{this.props.product.variantType}</div>
+              </div>
+              <div>
+                <div className="price-tag-container">
+                  <div className="price-tag">$</div>
+                  <div className="price-tag">{Math.floor(this.props.product.price)}</div>
+                  <div className="price-tag">{'.' + (Math.floor((this.props.product.price % 1) * 100) + '0').slice(-2)}</div>
+                </div>
+              </div>
+            </div>
+            <ReviewStars stars={this.props.product.averageRating} include={true} reviews={this.props.product.reviews}/>
+        </div>
+        <Stylepicker product={this.props.product} />
+        <div className="bottom-space flex">
+            <div className="cart-add">Add to cart</div>
+            <div className="cart-like">
+              <div>♡</div>
+            </div>
+        </div>
+        <div className="product-purchase-availability">
+            <div className="availability-delivery bottom-grey">
+              <div className="availability-svg">
```

Comment:
i like to import svg's like this, cleans the code and makes it clear what the svg is